package.json

```diff
@@ -4,6 +4,10 @@
   "description": "Search analytics by Algolia",
   "jsdelivr": "dist/search-insights.min.js",
   "main": "dist/search-insights.min.js",
+  "engineStrict": true,
```

Comment:
where's the documentation of this? I haven't seen it before

Reply:
Oh, now I checked again and it's removed a long time ago. I'm removing this.

Reply:
https://github.com/algolia/search-insights.js/pull/146/commits/553aec6a12047c46f899dae04d6d0398a41ae4fa